docs/mosip-installation-without-public-dns.md

```diff
@@ -46,48 +53,90 @@
               proxy_pass_request_headers      on;
               proxy_set_header  Strict-Transport-Security "max-age=0;";
          }
-         location / { return 301 https://iam.soil.mosip.net; }
+         location / { return 301 https://iam.sandbox.xyz.net; }
       }
   }
   ```
+  **Note:** We enable http access for IAM because MOSIP's keymanager expects to have valid ssl certificates.
```

Comment:
Avoid using we enable..... HTTP access is enabled for IAM because MOSIP's keymanager expects to have valid SSL certificates....  (please check if it is technically correct and update)